nipype/interfaces/niftyseg/stats.py

```diff
@@ -0,0 +1,213 @@
+        self.output = self._parse_stdout(new_runtime.stdout)
+        return new_runtime
+
+    def _list_outputs(self):
+        outputs = self.output_spec().get()
+        outputs['output'] = self.output
+        return outputs
+
+
+class UnaryStatsInput(StatsInput):
+    """Input Spec for seg_stats unary operations."""
+    operation = traits.Enum('r', 'R', 'a', 's', 'v', 'vl', 'vp', 'n', 'np',
+                            'e', 'ne', 'x', 'X', 'c', 'B', 'xvox', 'xdim',
+                            argstr='-%s',
+                            position=4,
+                            mandatory=True,
+                            desc='operation to perform')
+
+
+class UnaryStats(StatsCommand):
+    """
+    Interface for executable seg_stats from NiftySeg platform.
+
+    Interface to use any unary statistical operations that can be performed
+    with the seg_stats command-line program. See below for those operations:
+        -r          | The range <min max> of all voxels.
+        -R          | The robust range (assuming 2% outliers on both sides)
+                    | of all voxels
+        -a          | Average of all voxels
+        -s          | Standard deviation of all voxels
+        -v          | Volume of all voxels above 0 (<# voxels> *
+                    | <volume per voxel>)
+        -vl         | Volume of each integer label (<# voxels per label> *
+                    | <volume per voxel>)
+        -vp         | Volume of all probabilsitic voxels (sum(<in>) *
+                    | <volume per voxel>)
+        -n          | Count of all voxels above 0 (<# voxels>)
+        -np         | Sum of all fuzzy voxels (sum(<in>))
+        -e          | Entropy of all voxels
+        -ne         | Normalized entropy of all voxels
+        -x          | Location (i j k x y z) of the smallest value in the image
+        -X          | Location (i j k x y z) of the largest value in the image
+        -c          | Location (i j k x y z) of the centre of mass of the
+                    | object
+        -B          | Bounding box of all nonzero voxels
+                    | [ xmin xsize ymin ysize zmin zsize ]
+        -xvox       | Output the number of voxels in the x direction.
+                    | Replace x with y/z for other directions.
+        -xdim       | Output the voxel dimention in the x direction.
+                    | Replace x with y/z for other directions.
+
+    Note: All NaN or Inf are ignored for all stats.
+          The -m and -t options can be used in conjusction.
+
+    For source code, see http://cmictig.cs.ucl.ac.uk/wiki/index.php/NiftySeg
+    For Documentation, see:
+        http://cmictig.cs.ucl.ac.uk/wiki/index.php/NiftySeg_documentation
+
+    Examples
+    --------
+    >>> from nipype.interfaces import niftyseg
+    >>> node = niftyseg.UnaryStats()
+    >>> node.inputs.in_file = 'im1.nii'
+    >>> node.inputs.operation = 'v'
+    >>> node.cmdline  # doctest: +ALLOW_UNICODE
```

Comment:
I think all these interfaces would deserve some more doctests with more operations, so that we are sure that they work and that we don't break them in future maintenance.